src/registrar/admin.py

```diff
@@ -1239,6 +1239,29 @@ class DraftDomainAdmin(ListHeaderAdmin):
     search_help_text = "Search by draft domain name."
 
 
+class VeryImportantPersonAdmin(ListHeaderAdmin):
+    list_display = ("email", "requestor", "truncated_notes", "created_at")
+    search_fields = ["email"]
+    search_help_text = "Search by email."
+    list_filter = [
+        "requestor",
+    ]
+    readonly_fields = [
+        "requestor",
+    ]
+
+    def truncated_notes(self, obj):
+        # Truncate the 'notes' field to 200 characters
+        return str(obj.notes)[:50]
```

Comment:
```suggestion
    def truncated_notes(self, obj):
        # Truncate the 'notes' field to 50 characters
        return str(obj.notes)[:50]
```
Just noticed this now - doesn't this only truncate 50 characters?

Reply:
Totally. I started with 200 and it was too much.